test/integration/acceptance/custom/headless/headless.go

```diff
@@ -0,0 +1,390 @@
+package headless
+
+import (
+	"context"
+	"fmt"
+	"github.com/skupperproject/skupper/client"
+	"github.com/skupperproject/skupper/test/utils/tools"
+	apiv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/prometheus/common/log"
+	"github.com/skupperproject/skupper/api/types"
+	"github.com/skupperproject/skupper/pkg/kube"
+	"github.com/skupperproject/skupper/test/utils/base"
+	"github.com/skupperproject/skupper/test/utils/constants"
+	"gotest.tools/assert"
+)
+
+const (
+	appName     = "nginx"
+	appPort     = "8080"
+	appPortInt  = 8080
+	appPortName = "web"
+	appImage    = "nginxinc/nginx-unprivileged:stable-alpine"
+	pod0Name    = appName + "-0"
+	pod1Name    = appName + "-1"
+)
+
+type BasicTestRunner struct {
+	base.ClusterTestRunnerBase
+}
+
+func (r *BasicTestRunner) TestsHeadlessElements(ctx context.Context, t *testing.T) {
+
+	const (
+		pod0ProxyName = appName + "-proxy-0"
+		pod1ProxyName = appName + "-proxy-1"
+		svcProxyName  = appName + "-proxy"
+		timeout       = 120 * time.Second
+		interval      = 5 * time.Second
+	)
+
+	pubCluster, err := r.GetPublicContext(1)
+	assert.Assert(t, err)
+
+	prvCluster, err := r.GetPrivateContext(1)
+	assert.Assert(t, err)
+
+	assert.Assert(t, base.WaitForSkupperConnectedSites(ctx, pubCluster, 1))
+	assert.Assert(t, base.WaitForSkupperConnectedSites(ctx, prvCluster, 1))
+
+	t.Run("headless-inspect-pods", func(t *testing.T) {
+		t.Logf("Testing: %s\n", "All proxy pods got deployed")
```

Comment:
For this and the others like this, any reason for this format?  The second element is a literal, so why the %s substitution?